x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_reducer.ts

```diff
@@ -152,7 +152,11 @@ export const alertReducer = <AlertPhase extends InitialAlert | Alert>(
         keyof AlertAction,
         SavedObjectAttribute
       >;
-      if (index === undefined || isEqual(alert.actions[index][key], value)) {
+      if (
+        index === undefined ||
+        alert.actions.length === 0 ||
```

Comment:
this check makes sure we don't try to update subparams when the action does not exist